ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/LoggingHandler.java

```diff
@@ -87,9 +88,10 @@ public void handle(final RoutingContext routingContext) {
       final String userAgent = Optional.ofNullable(
           routingContext.request().getHeader(HTTP_HEADER_USER_AGENT)).orElse("-");
       final String timestamp = Utils.formatRFC1123DateTime(clock.millis());
+      final SocketAddress socketAddress = routingContext.request().remoteAddress();
       final String message = String.format(
           "%s - %s [%s] \"%s %s %s\" %d %d \"-\" \"%s\" %d",
-          routingContext.request().remoteAddress().host(),
+          socketAddress == null ? "null" : socketAddress.host(),
```

Comment:
Not sure why, but I got an NPE on this line a couple of times. This should prevent it.